src/index.js

```diff
@@ -20,15 +20,82 @@ function createElement(tagName, props, ...children) {
   return element;
 }
 
-function render() {
+function render(display, calculator) {
+  const numbers = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9];
+  const operators = [
+    { label: '+', func: (number1) => (number2) => number1 + number2 },
+    { label: '-', func: (number1) => (number2) => number1 - number2 },
+    { label: '*', func: (number1) => (number2) => number1 * number2 },
+    { label: '/', func: (number1) => (number2) => number1 / number2 },
+    { label: '=', func: () => null },
+  ];
+
+  const handleNumberClick = (event) => {
```

Comment:
```suggestion
  const handleNumberClick = (value) => {
```
로 받는다면 굳이 event.target으로 DOM에서 값을 꺼내올 필요가 없죠.